### PR TITLE
Add ability to delay the message

### PR DESCRIPTION
**Pull request by mbseid**
Takes advantage of the setDelaySeconds function of the SendMessageRequest. 

Added some overloads of the send message function to allow for easy access of sending delayed messages. Tested and works great.

### CODE REVIEW

File: src/main/scala/com/kifi/franz/FakeSQSQueue.scala [hunk 1]
```diff
@@ -17,7 +17,7 @@ trait FakeSQSQueue[T] extends SQSQueue[T] {
 
   override def initQueueUrl(): String = ""
 
-  override def send(msg: T, messageAttributes: Option[Map[String,String]]): Future[MessageId] = Future.successful(MessageId(""))
+  override def send(msg: T, messageAttributes: Option[Map[String,String]], delay: Option[Int]): Future[MessageId] = Future.successful(MessageId(""))
```
Comment (stkem) [inline src/main/scala/com/kifi/franz/FakeSQSQueue.scala]: Would you mind making this default to `None` as well to preserve signature compatibility?

**Comment by stkem** [comment]
Nice!

File: src/main/scala/com/kifi/franz/FakeSQSQueue.scala (hunk 1 above)
Comment (mbseid) [inline src/main/scala/com/kifi/franz/FakeSQSQueue.scala]: Most definitely. Will update it now.

**Comment by mbseid** [comment]
Updated and ready to go. Thanks for the feedback!

**Comment by mbseid** [comment]
Thanks! Any chance this is published? I left the build settings as 0.3.12-SNAPSHOT, but can be released as 0.3.12 if you want. Let me know if you need any help, happy to do as much as I can. 

Thanks again for the awesome library.

**Comment by stkem** [comment]
Waiting on another open PR to get finished and then I'll publish another version. Should be in the next few days. Feel free to nag if that doesn't happen :-)
Glad you like the library.

**Comment by mbseid** [comment]
Thanks for the update! Looking forward to it.

**Pull request merged**